DEPLOYMENT.md

```diff
@@ -2,6 +2,32 @@
 
 Host on Heroku. [Integrated with GitHub](https://devcenter.heroku.com/articles/github-integration).
 
+## Prepare your environment
+
+```
+cd your_projects_folder
+git clone https://github.com/rubytaiwan/rubytw-reboot.git
+heroku login
+heroku git:remote -a rubytw-staging -r staging
+heroku git:remote -a rubytw-production -r production
+git remote -v
+# you will got staging/production remote name now.
+```
+
+## Local development environment
```

Comment:
Sorry, I missed the bin/setup script, so this content will be remove.
thank you.